library/Ivoz/Provider/Domain/Model/Company/CompanyDto.php

```diff
@@ -101,7 +103,6 @@ public function normalize(string $context, string $role = ''): array
             $response['codecIds'] = $this->codecIds;
             $response['accountStatus'] = $this->accountStatus;
             $response['currentDayMaxUsage'] = $this->currentDayMaxUsage;
-            $response['accountStatus'] = $this->accountStatus;
```

Comment:
Why has this been removed? I can't find any description indicating this should be removed.

Reply:
Duplicated line. Look two lines above.